src/Core/Scene/Scene.cpp

```diff
@@ -493,8 +493,11 @@ namespace obe::Scene
     void Scene::removeGameObject(const std::string& id)
     {
         m_gameObjectArray.erase(std::remove_if(m_gameObjectArray.begin(), m_gameObjectArray.end(), [&id](const std::unique_ptr<Script::GameObject>& ptr){
-            if (ptr->getId() == id)
+            if (ptr->getId() == id){
```

Comment:
Maybe `return ptr->getId() == id;` would be a cleaner way to do it ?

Reply:
Good idea! I will modify it